Victor_Corado/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
```

Comment:
The name of your folder should be `victor_corado`

Reply:
folder rename fixed